scripts/src/sound.lua

```diff
@@ -62,6 +62,20 @@ end
 
 
 
+---------------------------------------------------
+-- Super A'Can
+--@src/devices/sound/acan.h,SOUNDS["ACAN"] = true
+---------------------------------------------------
+
+if (SOUNDS["ACAN"]~=null) then
+	files {
+		MAME_DIR .. "src/devices/sound/acan.cpp",
+		MAME_DIR .. "src/devices/sound/acan.h",
```

Comment:
Please move these to src/mame/audio and add them to the `funtech` target in scripts/target/mame/mess.lua  since they’re essentially a one-off.